package.json

```diff
@@ -40,23 +40,23 @@
     "@types/react": "^18.0.0",
     "@types/react-native": "^0.69.1",
     "@types/react-test-renderer": "^18.0.0",
-    "babel-jest": "^28.0.0",
+    "babel-jest": "^29.0.0",
     "conventional-changelog-cli": "^2.0.11",
     "cp-cli": "^2.0.0",
     "dedent": "^0.7.0",
     "del-cli": "^5.0.0",
     "eslint": "^8.21.0",
     "flow-bin": "~0.170.0",
     "flow-copy-source": "^2.0.9",
-    "jest": "^28.0.0",
+    "jest": "^29.0.0",
     "react": "^18.0.0",
     "react-native": "~0.69.1",
     "react-test-renderer": "^18.0.0",
     "strip-ansi": "^6.0.0",
     "typescript": "^4.0.2"
   },
   "dependencies": {
-    "pretty-format": "^28.1.3"
+    "pretty-format": "^29.0.0"
```

Comment:
@thymikee could you release this? 😀

Reply:
Lemme double check with @mdjastrzebski about the preferred next version and we'll release, most likely tomorrow :) Thanks for the nudge!